### PR TITLE
chore!: rename tuples to tasks

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Asana

https://app.asana.com/0/1203124674173179/1203842723899063/f

## Companion PRs

- https://github.com/Substra/substra-tests/pull/239
- https://github.com/Substra/substra/pull/337
- https://github.com/Substra/substrafl/pull/79
- 
## E2E tests

- https://github.com/owkin/substra-ci/actions/runs/4044324704

### CODE REVIEW

File: references/sdk_models.md [hunk 1]
```diff
@@ -23,7 +23,6 @@ Data sample
 - data_manager_keys: Optional[List[str]]
 - path: Optional[DirectoryPath]
 - creation_date: datetime
-- test_only: bool
```
Comment (oleobal) [inline references/sdk_models.md:26]: Is this left over from something else?

File: references/sdk_models.md [hunk 2]
```diff
@@ -34,8 +33,7 @@ Dataset asset
 - owner: str
 - permissions: Permissions
 - type: str
-- train_data_sample_keys: List[str]
-- test_data_sample_keys: List[str]
+- data_sample_keys: List[str]
```
Comment (oleobal) [inline references/sdk_models.md:36]: Is this a generic task change?

File: references/sdk_models.md (hunk 1 above)
Comment (ThibaultFy) [reply references/sdk_models.md:26]: Wooops this is a rebase issue I think

Comment (ThibaultFy) [reply references/sdk_models.md:26]: Ah no this is due to the make doc command, that was not done in the previous PR... On substra, you can skip the .md files in the review as they are automatically generated

**Review by oleobal — APPROVED** [review]

**Pull request merged**